### ISSUE TITLE
webapp.bash has to check for pod status before forwarding port.

### ISSUE THREAD

**Issue by amsaha** [CiscoAI/KFLab#102]
The following timeout based approach does not work reliably:
```
# sleep till the svc comes up
echo "Wait till the client svc comes up:"
sleep 30

#port forward mnist client port to a local port
kubectl -n ${NAMESPACE} port-forward svc/tf-mnist-client ${WEBAPP_PORT}:80 
```

**Comment by amsaha** [comment]
Also, we need a message at the end stating the IP:port. Such a message was earlier present at 
https://github.com/CiscoAI/KFLab/blob/master/tf-mnist/webapp.bash#L40 but it was removed. 

Also, why is the port fixed at 80 now whereas earlier we were getting node_port by running
```
#NODE_PORT=`kubectl get svc/tf-mnist-client -n ${NAMESPACE} -o jsonpath='{.spec.ports[0].nodePort}'`
```
?

**Comment by krishnadurai** [comment]
The port currently mentioned is the port on the container (80) which is being directly exposed to our local machine's port (9001).
The node port way of accessing the container port does not involve any forwarding to our local machine whereas it forwards the container port to the node on which the container is running on. This is one of making the port available to systems outside of k8s cluster.

**Comment by amsaha** [comment]
Great! Thanks for the explanation. Closed via pull #109.
